### DISCUSSION TITLE
[BUG]9p 算力 已经 16天没出块了。

### DISCUSSION THREAD

**Discussion by qqguaer** [Chia-Network/chia-blockchain#8593] [Support]
[BUG]9p 算力 已经 16天没出块了。
  检查了所有日记 都没问题  就是不出块
请问我钱包是不是被官方拉黑了？

**Comment by loppefaaret** [comment]
Even if someone has control of your private key, they would not be able to stop your farm from creating blocks.
I dont know if the chinese government is able to hack the chia client, and stop it from creating blocks - but i doubt they would.
perhaps you are just unlucky, and will get a block soon ? 
make sure your farming is running good, that you dont have long scan times on your harvesters, and that the farmer is in sync

**Comment by BrandtH22** [comment]
@qqguaer , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
